test/functional/tests/fault_injection/test_fault_injection_with_mounted_core.py

```diff
@@ -166,3 +166,39 @@ def test_add_occupied_core(cache_mode):
 
     with TestRun.step("Stop all caches."):
         casadm.stop_all_caches()
+
+
+@pytest.mark.parametrize("cache_mode", CacheMode)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_add_twice_same_core(cache_mode):
+    """
+        title: Fault injection test if cache not accepts twice the same device as core.
+        description: |
+          Negative test of the ability to add the same core twice to cache.
+        pass_criteria:
+          - No system crash.
+          - The same core device cannot be used twice in one cache.
```

Comment:
If I'm correct it actually shouldn't be possible to be used twice even in different caches